examples/SoundClassification/index.html

```diff
@@ -0,0 +1,21 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Sound classification using SpeechCommands18w</title>
+    <script>
+        // Required by some npm packages
+        window.process = { browser: true, env: { ENVIRONMENT: 'BROWSER' } };
+    </script>
+    <script src="../../dist/ml5.js"></script>
+</head>
+
+<body>
+    <h1>Sound classification using SpeechCommands18w</h1>
+    <p>Try to speak the following commands to your microphone: 'zero', 'one', 'two', 'three', 'four', 'five', 'six',
+        'seven', 'eight', 'nine', 'up', 'down', 'left', 'right', 'go', 'stop', 'yes', and 'no', in addition to
+        'background_noise' and 'unknown'</p>
+    <script src="sketch.js"></script>
+</body>
+
+</html>
```

Comment:
FYI: this symbol (and the associated warning you might see when committing) comes from Git expecting that there's a new-line character at the end of each file.

Our repo has `prettier`, which can take care of such minor issues. You could run `npm run format` and then just commit the changes pertaining to your code to the branch.